src/main/c/javacan_socketcan.c

```diff
@@ -41,6 +41,14 @@ JNIEXPORT jint JNICALL Java_tel_schich_javacan_SocketCAN_createRawSocket(JNIEnv
     return fd;
 }
 
+JNIEXPORT jint JNICALL Java_tel_schich_javacan_SocketCAN_createBcmSocket(JNIEnv *env, jclass class) {
+    jint fd = create_can_bcm_socket();
+    if (fd == -1) {
+    	throw_native_exception(env, "Unable to create BCM socket");
```

Comment:
nitpick: something with the formatter seems to be off

Reply:
My formatter is a best guess... Can you provide a reference to the settings you use? Or export your settings and send it to me.

Reply:
I'm applying basically the same style as for Java. 4 space indention. I'm using CLion, it defaults to that style.

Reply:
okay, fixed.